test/AST-Quake/vector-ctor-initlist.cpp

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// Test for std::vector initializer_list constructor support
+
+// RUN: cudaq-quake %s | cudaq-opt --canonicalize | FileCheck %s
```

Comment:
The `--canonicalize` pass is a NOP now. Although just running the output through `cudaq-opt` will throw away the debug information for a smaller output.